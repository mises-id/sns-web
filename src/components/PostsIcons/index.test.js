/*
 * @Author: lmk
 * @Date: 2021-07-15 10:12:21
 * @LastEditTime: 2021-07-16 14:16:42
 * @LastEditors: lmk
 * @Description: PostsIcon test
 */
import React from 'react';
import {shallow} from 'enzyme';
import PostsIcon from './';
describe('PostsIcon', () => {
  it('should render correctly', () => {
    const component = shallow(<PostsIcon />)
    expect(component).toMatchSnapshot();
  });
});