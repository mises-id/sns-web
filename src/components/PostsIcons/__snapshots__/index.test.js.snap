// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostsIcon should render correctly 1`] = `
<div
  className="m-flex itemFunctionBox"
>
  <div
    className="m-flex"
    onClick={[Function]}
  >
    <img
      alt="like"
      className="iconStyle"
      src="like.png"
    />
    <span
      className="m-font12 m-margin-left8 m-colors-333"
    />
  </div>
  <div
    className="m-flex"
  >
    <img
      alt="comment"
      className="iconStyle"
      src="comment.png"
    />
    <span
      className="m-font12 m-colors-333 m-margin-left8"
    />
  </div>
  <div
    className="m-flex"
    onClick={[Function]}
  >
    <img
      alt="forward"
      className="iconStyle"
      src="forward.png"
    />
    <span
      className="m-font12 m-colors-333 m-margin-left8"
    />
  </div>
</div>
`;
